Added "Store DIP" choice in config page

jrwdunham opened the pull request:
This required also modifying the workflow (hence the migration) because
the post-DIP-upload "Store DIP?" choice link was being activated AFTER
relocation of the DIP to the uploadedDIPs/ directory, which resulted in
the MCPServer not knowing where the DIP was when it went to look for the
processingMCP.xml config file. Now moving the DIP to uploadedDIPs/ only
occurs after the upload/store DIP steps have completed.

sevein commented:
So no changes in `src/dashboard/src/components/ingest/views.py` other than cosmetics, right?

jrwdunham commented:
Yeah, views.py only has cosmetic changes.

sevein reviewed (changes requested):
Migration conflicts with `0034_add_readme_to_aips`.

jrwdunham committed:
Added "Store DIP" choice in config page

This required also modifying the workflow (hence the migration) because
the post-DIP-upload "Store DIP?" choice link was being activated AFTER
relocation of the DIP to the uploadedDIPs/ directory, which resulted in
the MCPServer not knowing where the DIP was when it went to look for the
processingMCP.xml config file. Now moving the DIP to uploadedDIPs/ only
occurs after the upload/store DIP steps have completed.

jrwdunham commented:
After discussion with Evelyn and Hector, we have agreed that this PR needs to be modified. The issue is that there are two distinct "Store DIP" paths, which means that processing configuration will only work for one of them. To avoid this, we propose to prune one of the paths. However, this requires additional work beyond just modifying the workflow. Choosing NOT to store a DIP after the "Upload DIP?" decision point has different implications depending on whether the user has or has not Uploaded the DIP: if she has not uploaded, then not storing will lead to rejection of the DIP; however if she has uploaded, then not storing will not lead to rejection. Future commits and CR will are forthcoming for this PR.

jrwdunham commented:
Closed in favour of https://github.com/artefactual/archivematica/pull/680.
